packages/components/addon/components/hds/form/text-input/field.hbs

```diff
@@ -0,0 +1,18 @@
+<div class={{this.classNames}}>
+  {{yield (hash Label=(component "hds/form/label" controlId=this.id _contextualClass="hds-form-field__label"))}}
+  {{yield
+    (hash
+      HelperText=(component "hds/form/helper-text" controlId=this.id _contextualClass="hds-form-field__helper-text")
+    )
+  }}
+  <Hds::Form::TextInput::Base
+    @type={{@type}}
+    @value={{@value}}
+    @isInvalid={{@isInvalid}}
+    @width={{@width}}
+    ...attributes
+    id={{this.id}}
+    aria-describedby={{this.ariaDescribedBy}}
```

Comment:
[issue] the way this is currently implemented, an error will be thrown because it always include help text and error message text, even if there aren't any (therefore those elements wouldn't exist).

I wonder if you can try something like what we did in the [button element for aria-label](https://github.com/hashicorp/design-system/blob/6a7f752ce59ca284a16e6c48ff0a55f9c425bc48/packages/components/addon/components/hds/button/index.hbs#L12) or something similar? 

`aria-describedby="{{if this.hasHelpText this.helpTextId null}} {{if this.hasErrorText this.errorTextId null}}"`

Reply:
@alex-ju opened a Asana ticket specifically for this issue: https://app.asana.com/0/1202008719910219/1202443172675747